source/common/http/filter_manager.h

```diff
@@ -744,6 +745,16 @@ class FilterManager : public ScopeTrackedObject,
                               const std::function<void(ResponseHeaderMap& headers)>& modify_headers,
                               const absl::optional<Grpc::Status::GrpcStatus> grpc_status,
                               absl::string_view details) PURE;
+  /**
+   * Executes any prepared, but not yet propagated, local reply.
```

Comment:
```suggestion
   * Executes a prepared, but not yet propagated, local reply.
```

`any` makes it sound like there could be multiple, but there will only ever be one